### PR TITLE
updates to italian translations via @paolomi from. fixes #1165

### LINKED ISSUE #1165: fix for italian translation (ordinal numbers and other typos)
Hi :)
I've updated italian translation, in particular I've changed ordinal numbers from "1°"  (1th) to "prima" (first) and so on, because speech synthesizers read them as temperature. I hope the official italian translator can review my update. Thanks.
Ps: I'm a noob about git, so my translation is here in attachment.

[italian.zip](https://github.com/valhalla/valhalla/files/1844493/italian.zip)

### PR DESCRIPTION
**Pull request by kevinkreiser**
@paolomi put together some updates for the italian translations. one alteration i made to the original issues attachment was that i only applied the change in ordinal values to the verbal roundabout instructions. the text instructions retain the shorter `1°` etc as these are not meant for TTS engines anyway.

### CODE REVIEW

**Comment by paolomi** [comment]
@kevinkreiser  
I came here because I use "WhoGo Maps"(*) with "OSM Scout Server"(**) and there is the problem with shorter ordinal values: in italian they are spoken as temperature (ex: 1° C). You wrote that these are not meant for TTS engines, so do you suggest to contact OSM scout server's author to fix it? Thanks a lot.

(*) https://openrepos.net/content/rinigus/osm-scout-server
(**) https://openrepos.net/content/otsaloma/whogo-maps

**Comment by kevinkreiser** [comment]
@paolomi please note that i did keep your changes for the voice instructions, just not the text ones. i dont know about osm scout server or whogo maps but if they are passing the voice string to the TTS engine then this will work fine. notice that in our json we have two types of strings, there is always one marked `verbal` and one not. the verbal one is the one that they should use for TTS engines. if they arent doing that it should be very easy for them to start doing so. let me know if this doesnt make sense!

File: locales/it-IT.json [hunk 1]
```diff
@@ -1,1927 +1,1927 @@
...
+          "Entra nella rotonda e prendi la 6° uscita.",
+          "Entra nella rotonda e prendi la 7° uscita.",
+          "Entra nella rotonda e prendi la 8° uscita.",
+          "Entra nella rotonda e prendi la 9° uscita.",
           "Entra nella rotonda e prendi la 10° uscita."
         ]
       }
-    }, 
+    },
     "enter_roundabout_verbal": {
```
Comment (kevinkreiser) [inline locales/it-IT.json]: @paolomi notice the `verbal` on the end of this one? this is the one meant for TTS engines

Comment (paolomi) [reply locales/it-IT.json]: @kevinkreiser 
ah ok, now I've understood! :-)  It's ok! many thanks! :-)

Comment (rinigus) [reply locales/it-IT.json]: @paolomi, we use verbal strings for TTS, so everything should be fine then. 

I would like to use a chance and thank everyone behind Valhalla - it's a great project and we are using it as a part of full offline navigation solution on Sailfish OS (other components include MapboxGL and libpostal). Thank you very much and good luck with the new endeavor at Mapbox!

Comment (kevinkreiser) [reply locales/it-IT.json]: thank you so much @rinigus we're very happy to hear that everything worked out with your project!

**Pull request merged**